python/ql/test/library-tests/PointsTo/calls/GetACall.ql

```diff
@@ -1,5 +1,5 @@
 import python
 
-from ControlFlowNode call, FunctionObject func
+from ControlFlowNode call, Value func
```

Comment:
Is there a reason you're using `Value` instead of `CallableValue`?

Reply:
Yep, I wanted to capture that some `Value`s can have a call without being a `CallableValue` -- classes is the only example that comes to mind.